### ISSUE TITLE
Total Requests per Hour

### ISSUE THREAD

**Issue by vectortowns** [lgapontes/hey-joe#22]
I think "Total Requests per Hour" is wrong. Sometimes it registers 2 hits instead of 1.

**Comment by lgapontes** [comment]
Checking...

**Comment by lgapontes** [comment]
The problems of the variable "requests" had already been fixed in version 0.1.0. Let's review some tests and confirm that the reported error no longer exists. In future versions we will make test classes available for you to do when necessary. By retreating the variables, we discovered other errors:

- There was a chart problem of the "requestsMeanTime" variable, where the maximum value was not displayed correctly.
- There was a difficulty installing the dependencies associated with the "disk". The data collection library has been changed.
- Not all files downloaded were calculated correctly in "kbytesPerMinute". Instead of implementing the filter, we are using the "express-end" library to resolve this error.

All of these errors were fixed in version 0.1.1. I hope you enjoy...
